### ISSUE TITLE
Runtime error about obj_cleanup_defer

### ISSUE THREAD

**Issue by kemege** [cowboy-coders/nim-openssl-evp#3]
Programs containing openssl_evp doesn't run properly.

The compiling process seemed to be fine, but when I run the executable, it shows an error of 

> could not import: obj_cleanup_defer

This happened on both Windows 7 64-bit and Debian Wheezy.

I did find `extern int obj_cleanup_defer` in `objects.h` of OpenSSL, but was unable to import it: I tried to build a C program with `#include "openssl/objects.h", which gives me an undefined reference of obj_cleanup_defer.

I'm quite new to OpenSSL, so I wonder if this integer is really useful when using OpenSSL. If it's not, perhaps we can remove it from the code to avoid errors?

**Comment by jovial** [comment]
Seems to be present in my libcrypto.so:

```
nm -D /usr/lib/libcrypto.so | grep obj_cleanup_defer
```

gives:

```
0000000000475b70 B obj_cleanup_defer
```

Maybe this has been added recently? Like you, I am also not sure how useful it is. I will comment it out for now, and advise people to import when needed.

**Comment by kemege** [comment]
I checked again on my Debian, the nm command you provided simply gives no output.
The .so version seems to be 1.0.0 (according to the symbolic link name)

But on Windows, the binary I used are compiled from the 1.0.2 source code released on 2015-01-22, which I think should be recent enough.

Anyway, I can use the module with no problems at the moment. Maybe that's something for the OpenSSL guys to figure out.
